src/services/lease-service/index.ts

```diff
@@ -501,4 +501,41 @@ export const routes = (router: KoaRouter) => {
       }
     }
   )
+
+  /**
+   * Gets detailed information on a listings applicants
+   * Returns a list of all applicants on a listing by listing id
+   * Uses ListingId instead of rentalObjectCode since multiple listings can share the same rentalObjectCode for historical reasons
+   */
+  router.get('(.*)/listing/:listingId/applicants/details', async (ctx: any) => {
```

Comment:
ctx har redan en "inferred" `Koa.Context` (eller nåt i den stilen) typ.

```suggestion
  router.get('(.*)/listing/:listingId/applicants/details', async (ctx) => {
```